src/graph/validator/test/MutateValidatorTest.cpp

```diff
@@ -201,13 +201,13 @@ TEST_F(MutateValidatorTest, UpdateEdgeTest) {
     auto cmd = "UPDATE EDGE ON study \"Tom\"->\"Lily\" SET count = 1";
     ASSERT_FALSE(checkResult(cmd, {}));
   }
-  // Wrong expr "$^.peson.age"
+  // Wrong expr "$^.person_.age"
```

Comment:
Happy to drop, but I find it's easier to understand things are typos when they're explicit, and `peson` isn't particularly obvious...